Angular2-Starter-Project.md

```diff
@@ -0,0 +1,119 @@
+# React with Webpack File Structure
```

Comment:
I'm an **Angular** fanatic and i'm in war :hocho: against ~~React~~ :stuck_out_tongue_winking_eye: